editor/src/components/custom-code/__snapshots__/code-file.spec.ts.snap

```diff
@@ -1546,11 +1654,47 @@ export var App = (props) => {
       },
       "transpiledCode": "\\"use strict\\";
 
+var __createBinding = this && this.__createBinding || (Object.create ? function (o, m, k, k2) {
```

Comment:
where do these extra autogenerated functions come from?

Reply:
are these because the code now says `import * as React from 'react'` and for that it needs the importStar function?